api/server.go

```diff
@@ -601,7 +600,7 @@ func (s *Server) GetEventsForBlockIDs(_ context.Context, in *access.GetEventsFor
 			return nil, fmt.Errorf("could not get header at height %d: %w", height, err)
 		}
 
-		timestamp, err := ptypes.TimestampProto(header.Timestamp)
+		timestamp := timestamppb.New(header.Timestamp)
 		if err != nil {
```

Comment:
do we still need to check this error?

Reply:
right, removed the check. The IDE linter made these fixes